LogicalQ/Logical.py

```diff
@@ -1634,7 +1634,24 @@ def from_counts(cls, counts, n_logical_qubits, label, stabilizer_tableau, basis=
             for outcome_raw in outcomes_raw:
                 # @TODO - find a more reliable method that does not rely on the current indexing
                 # Get substring corresponding to logical measurement result
-                outcomes.append(outcome_raw[1:1+label[0]])
+                binary = ""
+                if all([char in ["0", "1"] for char in outcome_raw]):
+                    binary = outcome_raw
+                    # outcomes.append(outcome_raw[1:1+label[0]])
+                elif outcome_raw.startswith("0b"):
+                    binary = outcome_raw[2:]
+                    # outcomes.append(outcome_raw[3:3+label[0]])
+                elif outcome_raw.startswith("0x"):
+                    binary = str(bin(int(outcome_raw, 16)))[2:]
+                    # outcomes.append(binary[1:1+label[0]])
+                else:
+                    raise ValueError("Could not resolve count format")
+                if len(binary) < label[0]: # handling weird edge case
```

Comment:
In what scenario(s) does this edge case arise? It would be good to have it documented for future reference, maybe with a `@TODO` if it's really weird.